lib/externalGraderLocal.js

```diff
@@ -61,19 +61,23 @@ class Grader {
                 });
             },
             (callback) => {
-                docker.pull(question.external_grading_image, (err, stream) => {
-                    if (err) {
-                        logger.warn(`Error pulling "${question.external_grading_image}" image; attempting to fall back to cached version.`, err);
-                        return callback(null);
-                    }
+                if (config.externalGradingPullImagesFromDockerHub) {
```

Comment:
Nit: generally prefer to see something like this written as:

```js
if (!condition) {
  return callback(null);
}
doStuff();
```
instead of 
```js
if (condition) {
  doStuff();
} else {
    callback(null);
}
```
This keeps indentation from getting too deep and makes it immediately clear that the "not" case is really just doing nothing.